src/main.rs

```diff
@@ -23,6 +25,10 @@ struct Atdf2SvdOptions {
     #[options(free)]
     svd_path: Option<std::path::PathBuf>,
 
+    /// List of patches to apply.
+    #[options(short = "p", long = "patch")]
+    patches: Vec<String>,
```

Comment:
I'd prefer calling the option something like `--auto-patches` to make it explicit what this is about.

I feel like it is a bit confusing that the "default" patches are dropped as soon as this option is specified.  It also means we can't seamlessly add a new "default patch" in the future: Downstream users who explicitly pass the list of patches will not get the new one until they add it manually...

My suggestion would be: The default patches are always applied, like they are now.  The CLI option can then be used to enable additional ones where needed.

Reply:
Renamed it to --auto-patches and left the default patches always apply as before. Thanks for the review!